### DISCUSSION TITLE
Virtual Desktop Streamer Support (VR) on Atlas

### DISCUSSION THREAD

**Discussion by TrentonGB** [Atlas-OS/Atlas#1191] [Help & Support]
Hello, I'm trying to install the Virtual Desktop Streamer program to connect with my Meta Quest 3 and play Luke Ross's Cyberpunk 2077 mod in VR. The installer is failing with the error: "There is a problem with this Windows Installer package. A program required for this install to complete could not be run. Contact your support personnel or package vendor."

I am attaching the installer logs from the first attempt and then also after the second attempt after I manually ran VirtualDesktop.Service.msi as Chatgpt suggested. Chatgpt also suggests that this may be an issue with "local group policy settings or any software restriction policies in place on Atlas OS". I also checked with the Virtual Desktop Streamer discord and they suggested that it might be the windows media pack missing. 

[MSI3918.LOG](https://github.com/user-attachments/files/16651462/MSI3918.LOG)
[second attempt.LOG](https://github.com/user-attachments/files/16651463/second.attempt.LOG)

Edition	Windows 11 Pro
Version	23H2
Installed on	‎8/‎13/‎2024
OS build	22631.3958
Experience	Windows Feature Experience Pack 1000.22700.1026.0
i9-13900KF, RTX4090

**Comment by TrentonGB** [comment]
Minor update, I found that for that specific mod, I should actually not use Virtual Desktop. That said, I'm sure I'm not the only PC VR gamer who would like to have Virtual Desktop Streamer working and I have an almost totally clean install, just steam, google drive, the adobe suite, and chrome.

**Comment by Ast3risk-ops** [comment]
Seriously, just use ALVR. Cripes.

It's free and open source, virtual desktop is paid, so it's hard for anyone to test or use.
